tests/system_tests/client/can/test_python_can_kvaser.py

```
@@ -44,10 +44,16 @@ def teardown_method(self):
         """Clean up all tasks that were opened during test and close all connections."""
         self.can_interface_1.flush_tx_buffer()
         self.can_interface_2.flush_tx_buffer()
+        self.transport_interface_1.teardown_sync(True)
```

Ruff (0.16.2)

[warning] Boolean positional value in function call

(FBT003)